chore: update keto opl docs (#1745)
ory/docs: docs/keto/reference/ory-permission-language.mdx

# The "Ory Permission Language" Specification

Enforcing fine-grained permissions is a critical building block of mature technology solutions that protect privacy and identity
in the information age. Several proprietary languages to represent permission already exist, namely the

> [Authzed Schema Language](https://docs.authzed.com/reference/schema-lang) and [Auth0 FGA](https://docs.fga.dev/). Most permissions

Reviewer: ```suggestion
Authzed Schema Language and Auth0 FGA. Most permissions
```

we can name them but I would prefer not to link to them here

> are defined by normal developers who typically are most familiar with Web technologies like JavaScript or Typescript. We see the
> need for a developer-friendly configuration language for permissions that has such a small learning curve that most developers can
> understand and use it with close to no effort. We therefore chose to define our permissions configuration language as a subset of

Reviewer: ```suggestion
are defined by normal developers who typically are most familiar with Web technologies like JavaScript or Typescript. Ory recognizes the need for a configuration language for permissions with a short learning curve that developers can understand and use without effort. Because of this the Ory Permissions Language  is defined as a subset of
```

{}  repetition (0 to n times)
```

> Lower-case production names are used to identify lexical tokens. Non-terminals are in CamelCase. Lexical tokens are enclosed in

Reviewer: ```suggestion
Lowercase production names are used to identify lexical tokens. Non-terminals are in CamelCase. Lexical tokens are enclosed in
```

please revert, hyphen bad

double quotes `""` or single quotes `''`.

The form `a … b` represents the set of characters from a through b as alternatives. The horizontal ellipsis `…` is also used
elsewhere in the spec to informally denote various enumerations or code snippets that are not further specified.

> ## Configuraton text representation

Reviewer: ```suggestion
## Configuration text representation
```
